<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DClassifyManager</class>
 <widget class="QWidget" name="DClassifyManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DClassifyManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	outline:0px;
	color:#DCDCDC;
}

*:disabled{
	background:#444444;
	border-color:#484848;
}

QWidget{
	border:1px solid #DCDCDC;
	background:#484848;
	border-radius:0px;
	color:#DCDCDC;
	border-style:none;
}

QSplitter::handle{
background:#7c7c7c;
}

QAbstractButton{
	border-radius:0px;
	color:#DCDCDC;
	background-color: rgb(214, 214, 214);
	border-style:none;
}

QAbstractButton:hover{
	color:#FFFFFF;
	background-color:#00BB9E;
}

QAbstractButton:checked,QAbstractButton:pressed{
	color:#DCDCDC;
	border-style:solid;
	border-width:0px 0px 0px 2px;
	padding:4px 4px 4px 2px;
	border-color:#00BB9E;
	background-color:#444444;
}

QLineEdit{
	border:1px solid #242424;
	border-radius:3px;
	padding:2px;
	background-color: rgb(90, 90, 90);
	color:#DCDCDC;
	selection-background-color:#484848;
	selection-background-color: rgba(81, 163, 244);
}

QLineEdit:focus{
	border:1px solid #242424;
}

QLineEdit{
	lineedit-password-character:9679;
}

.QPushButton{
	border-style:none;
	border:1px solid #242424;
	color:#DCDCDC;
	padding:5px;
	min-height:15px;
	border-radius:5px;
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

.QPushButton:hover{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

.QPushButton:pressed{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

QRadioButton::indicator{
	width:15px;
	height:25px;
}

QRadioButton::indicator::unchecked{
	image:url(:/images/qss/psblack/radiobutton_unchecked.png);
}

QRadioButton::indicator::unchecked:disabled{	
	image:url(:/images/qss/psblack/radiobutton_unchecked_disable.png);
}

QRadioButton::indicator::checked{
	image:url(:/images/qss/psblack/radiobutton_checked.png);
}

QRadioButton::indicator::checked:disabled{
	image:url(:/images/qss/psblack/radiobutton_checked_disable.png);
}

QComboBox{
	border:1px solid #8f8f8f;
	background-color: rgb(118, 118, 118);
	border-radius: 5px;
}

QComboBox::down-arrow{
	image:url(:/images/qss/psblack/add_bottom.png);
	width:10px;
	height:10px;
	right:2px;
}

QComboBox::drop-down{
	subcontrol-origin:padding;
	subcontrol-position:top right;
	width:15px;
	border-left-width:0px;
	border-left-style:solid;
	border-top-right-radius:3px;
	border-bottom-right-radius:3px;
	border-left-color:#242424;
}

QComboBox::drop-down:on{
	top:1px;
}

QSlider::groove:horizontal{
	background:#484848;
	height:8px;
	border-radius:4px;
}

QSlider::add-page:horizontal{
	background:#484848;
	height:8px;
	border-radius:4px;
}

QSlider::sub-page:horizontal{
	background:#242424;
	height:8px;
	border-radius:4px;
}

QSlider::handle:horizontal{
	width:13px;
	margin-top:-3px;
	margin-bottom:-3px;
	border-radius:6px;
	background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #444444,stop:0.8 #242424);
}

QSlider::groove:vertical{
	width:8px;
	border-radius:4px;
	background:#484848;
}

QSlider::add-page:vertical{
	width:8px;
	border-radius:4px;
	background:#484848;
}

QSlider::sub-page:vertical{
	width:8px;
	border-radius:4px;
	background:#242424;
}

QSlider::handle:vertical{
	height:14px;
	margin-left:-3px;
	margin-right:-3px;
	border-radius:6px;
	background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #444444,stop:0.8 #242424);
}

QScrollBar:horizontal{
	background:#484848;
	padding:0px;
	border-radius:6px;
	max-height:12px;
}

QScrollBar::handle:horizontal{
	background:#525252;
	min-width:50px;
	border-radius:6px;
}

QScrollBar::handle:horizontal:hover{
	background:#242424;
}

QScrollBar::handle:horizontal:pressed{
	background:#242424;
}

QScrollBar::add-page:horizontal{
	background:none;
}

QScrollBar::sub-page:horizontal{
	background:none;
}

QScrollBar::add-line:horizontal{
	background:none;
}

QScrollBar::sub-line:horizontal{
	background:none;
}

QScrollBar:vertical{
	background:#484848;
	padding:0px;
	border-radius:6px;
	max-width:12px;
}

QScrollBar::handle:vertical{
	background:#525252;
	min-height:50px;
	border-radius:6px;
}

QScrollBar::handle:vertical:hover{
	background:#242424;
}

QScrollBar::handle:vertical:pressed{
	background:#242424;
}

QScrollBar::add-page:vertical{
	background:none;
}

QScrollBar::sub-page:vertical{
	background:none;
}

QScrollBar::add-line:vertical{
	background:none;
}

QScrollBar::sub-line:vertical{
	background:none;
}

QScrollArea{
	border:0px;
}

QCheckBox::indicator{
	width:13px;
	height:13px;
}

QCheckBox::indicator:unchecked{
	image:url(:/qss/psblack/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:disabled{
	image:url(:/qss/psblack/checkbox_unchecked_disable.png);
}


QTreeWidget::indicator{
padding:0px -3px 0px 3px;
}

QTreeWidget::indicator{
	width:13px;
	height:13px;
}

QTreeWidget::branch:closed:has-children{
	margin:4px;
	border-image:url(:/images/qss/psblack/branch_open.png);
}

QTreeWidget::branch:open:has-children{
	margin:4px;
	border-image:url(:/images/qss/psblack/branch_close.png);
}

QTreeWidget::indicator:unchecked{
	image:url(:/images/qss/psblack/checkbox_unchecked.png);
}

QTreeWidget::indicator:unchecked:disabled{
	image:url(:/images/qss/psblack/checkbox_unchecked_disable.png);
}

QTreeWidget::indicator:checked{
	image:url(:/images/qss/psblack/checkbox_checked.png);
}

QTreeWidget::indicator:checked:disabled{
	image:url(:/images/qss/psblack/checkbox_checked_disable.png);
}

QTreeWidget::indicator:indeterminate{
	image:url(:/images/qss/psblack/checkbox_parcial.png);
}

QTreeWidget::indicator:indeterminate:disabled{
	image:url(:/images/qss/psblack/checkbox_parcial_disable.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mWidgetHeader" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="mComboxClassNames"/>
      </item>
      <item>
       <widget class="QLineEdit" name="mLineEditClassName">
        <property name="placeholderText">
         <string> Class Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnClassColor">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnNewAClass">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBtnDeleteAItem">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mMainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="mMainTreeWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="mWidgetBottom" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="mLabelExport">
           <property name="text">
            <string>Export To:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mComboxExportTo">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="mLineEditExportName">
           <property name="placeholderText">
            <string>PyObjectName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mBtnStartExport">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
